### PR TITLE
Readme: add some more explainations

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,25 @@ Blog post: https://discuss.ardupilot.org/t/gsoc-2021-custom-firmware-builder/749
 This website uses the Flask library. Flask must be installed before use.  
 Directories: `ardupilot` must be within `base`, which must be in the same directory as `CustomBuild`.
 
+### Directory structure
+The ardupilot directory must be in the same directory as the CustomBuild directory.
+
+`╭─khancyr@pop-os ~/Workspace/ardupilot_custom_build
```
Comment (willpiper24) [inline README.md:17]: This formatting doesn't seem to render properly and is a little confusing anyway, so would probably be best to remove it.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,25 @@ Blog post: https://discuss.ardupilot.org/t/gsoc-2021-custom-firmware-builder/749
...
+ ╰─$ tree -d -L 1
+ .
+ ├── ardupilot
+ └── CustomBuild
+`
+
+Use `--basedir` to adjust the base directory, the default one is `base`.
+It is expected that you have an environement where ArduPilot can be built. Otherwise, see [https://ardupilot.org/dev/docs/building-setup-linux.html](https://ardupilot.org/dev/docs/building-setup-linux.html)
```
Comment (willpiper24) [inline README.md:25]: Typo: environment

**Review by willpiper24 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (khancyr) [reply README.md]: done

**Comment by khancyr** [comment]
rebase and corrected

**Review by tridge — APPROVED** [review]

**Comment by tridge** [comment]
already merged

**Pull request closed without merging**